### ISSUE TITLE
Add the lux footer

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/lockers_and_study_spaces#219]
### Current display

There is no footer for this application.

### Desired display

We'd like to have the default lux footer in this application.

### Acceptance criteria

- [x] The footer appears in the footer
- [x] The footer includes a link to https://accessibility.princeton.edu or https://accessibility.princeton.edu/help
- [x] I can successfully use the footer with Voice Over in Safari.  If any errors, make sure they are ticketed in the lux repo (or fixed in a recent version).
- [x] I have checked the footer using one of the following browser extensions and confirmed that it does not add any level A or level AA issues.  If any errors, make sure they are ticketed in the lux repo (or fixed in a recent version).
    * axe dev tools
    * Lighthouse
    * Wave

**Comment by kevinreiss** [comment]
Lack of the accessibility link in the footer is an issue.
